gateway/api/ray.py

```diff
@@ -176,7 +179,14 @@ def create_ray_cluster(
     if not cluster_data:
         cluster = get_template("rayclustertemplate.yaml")
         manifest = cluster.render(
-            {"cluster_name": cluster_name, "user_id": user.username}
+            {
+                "cluster_name": cluster_name,
+                "user_id": user.username,
+                "workers": program_config.workers,
+                "min_workers": program_config.min_workers,
+                "max_workers": program_config.max_workers,
+                "auto_scaling": program_config.auto_scaling,
```

Comment:
What happens if bad values are passed? For example, a user sets min=5 and max=2... what kind of error message would they see? Is that caught by Ray or do we need to catch it? (I haven't tried yet, but based on past experience users will pass all kinds of combinations of bad values when configuring...)

Reply:
I tried it with max=1, min=2.  Ray didn't care, it just created 1 worker.